---
title: "Supplementary Information for:"
output:
  pdf_document:
    fig_caption: false
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE)
library(magrittr)
load("../data/Supp_mat.Rdata")
fig_num <- 0
tab_num <- 0
ins_tab <- function(){
  .GlobalEnv$tab_num <- .GlobalEnv$tab_num + 1
  paste0("Table S", .GlobalEnv$tab_num, ".")
}
ins_fig <- function(){
  .GlobalEnv$fig_num <- .GlobalEnv$fig_num + 1
  paste0("Figure S", .GlobalEnv$fig_num, ".")
}

library(kableExtra)
usepackage_latex("booktabs")
usepackage_latex("longtable")
usepackage_latex("array")
usepackage_latex("multirow")
usepackage_latex("wrapfig")
usepackage_latex("float")
usepackage_latex("colortbl")
usepackage_latex("pdflscape")
usepackage_latex("tabu")
usepackage_latex("threeparttable")
usepackage_latex("threeparttablex")
usepackage_latex("ulem", "normalem")
usepackage_latex("makecell")

```

# Assessing responses of micro-eukaryotes biodiversity to the Great Acceleration: a paleolimnological view based on sedimentary DNA

François Keck, Laurent Millet, Didier Debroas, David Etienne, Didier Galop and Isabelle Domaizon

` `  
` `  

Corresponding author: Isabelle Domaizon

Email: isabelle.domaizon@inrae.fr

` `  
` `  

**This PDF file includes:**

- Methods S1 to S5
- Figures S1 to S6
- Tables S1 to S3
- Legend for Dataset S1
- SI References

**Other supplementary materials for this manuscript include the following:**

Dataset S1

\pagebreak
## Supplementary Information Text

**Methods S1: General precautions taken for sedimentary DNA analysis**

Aprés validation finale COPIER LES METHODES ICI: pour l'instant le texte est dans le Google Drive.

<!-- \pagebreak -->
<!-- ```{r, message = FALSE} -->
<!-- lakes_tab %>%  -->
<!--   ggplot(aes(x = Altitude, fill = `Trophic status`)) + -->
<!--     geom_histogram(position = 'stack', binwidth = 500) + -->
<!--     scale_fill_manual(values = c("Eutrophic" = "#006D2C", -->
<!--                                  "Meso-eutrophic" = "#2CA25F", -->
<!--                                  "Mesotrophic" = "#66C2A4", -->
<!--                                  "Oligo-mesotrophic" = "#99D8C9", -->
<!--                                  "Oligotrophic" = "#CCECE6", -->
<!--                                  "Unknown" = "grey70")) + -->
<!--   labs(fill = "Trophic status") + xlab("Altitude (m)") + ylab("Count") + -->
<!--   theme_bw() -->


<!-- ``` -->


<!-- SKIPED NUM Distribution of lake elevation and trophic status. -->


\pagebreak

![](../figures/supplementary_info/SI_FIG_bioinfo.pdf)

**`r ins_fig()`** Tracking of the number of DNA reads and OTUs during the filtering and data pre-processing steps.


\pagebreak
```{r, fig.height = 8}
tax_rich_dat <-
  com_merged_raw_list %>% 
  filter(INVENTORY == "OTU") %>% 
  .$data %>% .[[1]] %>%
  spread_cdm(LAC_POS, TAXON, COUNT) %>% 
  rrarefy(sample = min(rowSums(.))) %>% 
  tidy_cdm(row.name = "LAC_POS") %>% 
  left_join(lakes_meta_nr)

tax_rich_lakes <- tax_rich_dat %>% 
  group_by(LAC, POSITION) %>% 
  summarise(RICH_LAKE = sum(COUNT > 0))

p <- tax_rich_dat %>% 
  left_join(otu_meta, by = c("TAXON" = "OTU_ID")) %>% 
  group_by(ADL_RANK_2, POSITION) %>% 
  nest() %>% 
  mutate(RICH = map(data, function(data){
    data %>% 
      spread_cdm(LAC, TAXON, COUNT) %>% 
      specnumber() %>% 
      enframe("LAC", "RICHNESS")
  })) %>% 
  unnest(RICH) %>% 
  left_join(tax_rich_lakes) %>% 
  mutate(REL_RICHNESS = RICHNESS/RICH_LAKE) %>% 
  group_by(ADL_RANK_2, POSITION) %>% 
  summarise(REL_RICHNESS_MEAN = mean(REL_RICHNESS),
            REL_RICHNESS_SD = sd(REL_RICHNESS)) %>% 
  left_join(read_csv("../data/Ranks_Adl.csv"), by = c("ADL_RANK_2" = "RANK2")) %>% 
  group_by(RANK1) %>% 
  mutate(GRP2_REL_RICHNESS_SUM = sum(REL_RICHNESS_MEAN, na.rm = TRUE)) %>%
  ungroup() %>% 
  mutate(ADL_RANK_2 = fct_explicit_na(ADL_RANK_2, "Unclassified"),
         RANK1 = fct_explicit_na(RANK1, "Unclassified")) %>% 
  ggplot() +
  geom_col(aes(fct_reorder(ADL_RANK_2, REL_RICHNESS_MEAN),
               REL_RICHNESS_MEAN, fill = POSITION),
           position = "dodge2") +
  facet_wrap(~ fct_reorder(RANK1, GRP2_REL_RICHNESS_SUM, .desc = TRUE),
             scales = "free_y", ncol = 1, strip.position = "left") +
  geom_linerange(aes(x = fct_reorder(ADL_RANK_2, REL_RICHNESS_MEAN),
                         ymin = REL_RICHNESS_MEAN + REL_RICHNESS_SD,
                         ymax = REL_RICHNESS_MEAN - REL_RICHNESS_SD),
           position = position_dodge2(width = 1, padding = 0.5)) +
  coord_flip() +
  scale_y_continuous(labels = scales::percent) +
  scale_fill_discrete(name = "Stratum", labels = c("Past (bottom)", "Recent (top)")) +
  xlab("") + ylab("Relative richness") +
  theme(strip.text.y = element_text(angle = 180),
        strip.placement = "outside")

gp <- ggplotGrob(p)
f_row <- gp$layout$t[grepl("panel", gp$layout$name)]
x_var <- sapply(ggplot_build(p)$layout$panel_scales_x,
                function(l) length(l$range$range))
gp$heights[f_row] <- gp$heights[f_row] * x_var
grid::grid.draw(gp)
```

**`r ins_fig()`**  Mean relative richness (number of distinct OTUs) for each taxa computed for the bottom and the top rarefied samples across all lakes. Horizontal black lines show the standard error.




<!-- \pagebreak -->
<!-- ```{r, fig.height = 7} -->

<!-- prow <- plot_grid(frac_taxa_temp$plots[[1]] + -->
<!--             theme(legend.position = "none"), -->
<!--           frac_taxa_temp$plots[[2]] + -->
<!--             theme(legend.position = "none") + -->
<!--             xlab(""), -->
<!--           labels = "AUTO") -->
<!-- legend <- get_legend( -->
<!--   frac_taxa_temp$plots[[1]] + -->
<!--     guides(color = guide_legend(nrow = 1)) + -->
<!--     theme(legend.position = "bottom", -->
<!--           legend.title = element_blank()) -->
<!-- ) -->
<!-- plot_grid(prow, legend, ncol = 1, rel_heights = c(1, .1)) -->
<!-- ``` -->

<!-- **`r `**  Proportion of OTUs (*A*) and taxonomic groups (*B*) specific to the bottom (Bottom only) and top (Top only) strata, or common to both strata (Intersection). The total number of OTUs or taxa is given for each lake in parenthesis. -->



\pagebreak
```{r, fig.height = 6}

SUPP_MAT$JACC$FIG_COMB

```


**`r ins_fig()`** Community analyses of the OTUs table reproduced with the Jaccard dissimilarity index. (*A*) NMDS of community compositions of the ‘recent’ (blue triangles) and ‘past’ (red circles) samples with 95% confidence ellipses represented for each group. Grey lines connect ‘recent’ and ‘past’ samples from the same lake. (*B*) Distribution of Jaccard index values computed for each lake between ‘recent’ and ‘past’ samples . (*C*) Distribution of the distances between samples and group geometric median for ‘recent’ (blue) and ‘past’ (red) samples. (*D*) Relationship between lake altitude and community dissimilarity (Jaccard index) between the ‘recent’ and ‘past’ strata. Fitted regression tree model is represented by black lines (mean values). Grey shading represent the 95% confidence intervals around means.


\pagebreak
```{r, fig.height = 6}

SUPP_MAT$TAXO$FIG_COMB

```

**`r ins_fig()`** Community analyses reproduced with the data aggregated at the 2^nd^ taxonomic rank of Adl et al. (21). (*A*) NMDS of community compositions of the ‘recent’ (blue triangles) and ‘past’ (red circles) samples with 95% confidence ellipses represented for each group. Grey lines connect ‘recent’ and ‘past’ samples taken within the same lake. (*B*) Distribution of Bray-Curtis index values computed for each lake between ‘recent’ and ‘past’ samples. (*C*) Distribution of the distances between samples and group geometric median for ‘recent’ (blue) and ‘past’ (red) samples. (*D*) Relationship between lake altitude and community dissimilarity (Bray-Curtis index) between the ‘recent’ and ‘past’ strata. Fitted regression tree model is represented by black lines (mean values). Grey shading represent the 95% confidence intervals around means.

\pagebreak
```{r, fig.height = 8}
deseq_TB_AOV1 %>%
  filter(INVENTORY == "Taxonomie Rank 3") %>%
  .$barplots %>% .[[1]] +
  theme(plot.title = element_blank())
```

**`r ins_fig()`** Magnitude of change (expressed in log2 fold change) in data aggregated at the 3^rd^ taxonomy rank of Adl et al. (21) between the past and recent strata, as estimated by the DESeq2 analysis (22). Dark bars represent groups for which the change was found significant (corrected p-value < 0.05). Horizontal lines show the standard error. The detailed results of the analysis are given in *SI Appendix*, Table S3.



\pagebreak
```{r, fig.height = 8}
t <- otu_tbl_replicates %>% 
  filter(!LAC %in% c("Léman", "Bourget")) %>% 
  mutate(ID_SAMPLE = str_c(LAC_CODE, POSITION, REPLICATE, sep = "_")) %>% 
  spread_cdm(ID_SAMPLE, OTU_ID, OTU_COUNT) %>% 
  vegdist(method = "bray") %>% 
  hclust() %>% 
  ape::as.phylo.hclust()
t2 <- phytools::splitTree(t, list(node = 188, bp = 0))
par(mfrow = c(1, 2))
ape::plot.phylo(ape::drop.tip(t, t2[[1]]$tip.label), cex = 0.5, no.margin = TRUE)
ape::plot.phylo(ape::drop.tip(t, t2[[2]]$tip.label), cex = 0.5, no.margin = TRUE)
par(mfrow = c(1, 1))
```

**`r ins_fig()`** Hierarchical clustering of the OTU matrix (Bray Curtis dissimilarity) including the technical replicates. The code of each sample corresponds to the combination of the lake (three-letter code; see *SI Appendix*, Table S1), stratum (top or bottom) and replicate number (1 or 2). Lakes Bourget and Léman are not represented as they have only one replicate. For better readability, the tree has been divided in two parts.



<!-- TABLE SECTION STARTS HERE -->

\pagebreak
```{r, message = FALSE}
lakes_kml <- lakes_kml[lakes_kml$Name %in% otu_merge_tbl$LAC, ]

lakes_tab <- read_csv("../data/lakes_coring.csv", na = character()) %>% 
  set_names(c("Code", "Name", "Longitude", "Latitude", "Altitude", "Depth\n(max)",
              "Trophic status", "Coring\nyear", "XrF", "$^{14}$C", "Radionuclides", "included_DNA")) %>% 
  mutate(Radionuclides = str_replace(Radionuclides, "137Cs", "$^{137}$Cs"),
         Radionuclides = str_replace(Radionuclides, "210Pb", "$^{210}$Pb")) %>% 
  arrange(included_DNA, Altitude)

not_included_min <- min(which(lakes_tab$included_DNA == 2))
not_included_max <- max(which(lakes_tab$included_DNA == 2))

lakes_tab <- select(lakes_tab, -included_DNA)

kableExtra::kable(lakes_tab, "latex", digits = 3, booktabs = TRUE, linesep = "", escape = FALSE,
                  col.names = linebreak(colnames(lakes_tab), align = "c")) %>% 
  kable_styling(latex_options = c("scale_down")) %>% 
  pack_rows("Not included for DNA analyses", not_included_min, not_included_max, indent = FALSE, latex_gap_space = "0.5em")

rm(not_included_min, not_included_max)

```

**`r ins_tab()`** List of the lakes included in the study with their three-letter code, geographic coordinates (decimal degrees; WGS84), altitude (meters) and maximum depth (meters). The trophic status, when referenced, is based on 3 parameters used in the Water Framework Directive, i.e. Secchi disk value, phosphorus concentration in the water and chlorophyll *a* concentration in the euphotic zone.


\pagebreak
```{r, fig.height = 6}
deseq_TB_AOV1 %>% 
  filter(INVENTORY == "Trophic Type") %>% .$results %>% .[[1]] %>%
  mutate(Taxa = str_replace(Taxa, "<NA>", "Unclassified (NA)")) %>%
  select(-lfcSE, -pvalue) %>% 
  knitr::kable("latex", digits = 3, booktabs = TRUE, linesep = "", escape = FALSE,
               col.names = linebreak(c("Trophic Group", "Base Mean",
                                       "log2 Fold Change", "Stat", "Padj"), align = "c")) %>% 
  kable_styling(position = "center")
```


**`r ins_tab()`** Detailed results of the DESeq2 analysis conducted on data aggregated by trophic groups. For each group, column *Base Mean* indicates the average of the normalized count values, dividing by size factors, *log2 Fold Change* is the magnitude of change between the past and recent strata, column *Stat* is the Wald statistic and *Padj* the p-value of the test corrected with the Benjamini & Hochberg method.



\pagebreak
```{r, fig.height = 6}
deseq_TB_AOV1 %>% 
  filter(INVENTORY == "Taxonomie Rank 3") %>% .$results %>% .[[1]] %>%
  left_join(select(otu_taxo_adl_conv, ADL_RANK_1, ADL_RANK_2, ADL_RANK_3), by = c("Taxa" = "ADL_RANK_3")) %>% 
  filter(!duplicated(Taxa)) %>% 
  select(Rank_1 = ADL_RANK_1, Rank_2 = ADL_RANK_2, Rank_3 = Taxa, baseMean, log2FoldChange, stat, padj) %>% 
  arrange(Rank_1, Rank_2, Rank_3) %>% 
  mutate(Rank_1 = ifelse(is.na(Rank_1), "Unclassified", Rank_1),
         Rank_2 = ifelse(is.na(Rank_2), "Unclassified", Rank_2)) %>% 
  mutate(Rank_1 = ifelse(!duplicated(Rank_1), Rank_1, ""),
         Rank_2 = ifelse(!duplicated(Rank_2), Rank_2, "")) %>% 
  knitr::kable("latex", digits = 3, booktabs = TRUE, linesep = "", escape = FALSE, longtable = TRUE,
               col.names = linebreak(c("Rank 1", "Rank 2", "Rank 3", "Base Mean",
                                       "log2 Fold\nChange", "Stat", "Padj"), align = "c")) %>% 
  add_header_above(c("Taxonomic rank (Adl et al. 2019)" = 3, " " = 4)) %>% 
  kable_styling(latex_options =c("repeat_header"))

```


**`r ins_tab()`** Detailed results of the DESeq2 analysis conducted on data aggregated at the 3^rd^ taxonomy rank of Adl et al. (21). Ranks 1 and 2 are given for guidance. For each taxa, column *Base Mean* indicates the average of the normalized count values, dividing by size factors, *log2 Fold Change* is the magnitude of change between the past and recent strata, column *Stat* is the Wald statistic and *Padj* the p-value of the test corrected with the Benjamini & Hochberg method.





\pagebreak

**Dataset S1 (separate file)** Complete list of OTUs used in the analyses with their representative DNA sequence, their taxonomic classification and their trophic classification. 

\pagebreak

## SI References


1.   A. C. Boere, W. I. C. Rijpstra, G. J. De Lange, J. S. Sinninghe Damsté, M. J. L. Coolen, Preservation potential of ancient plankton DNA in Pleistocene marine sediments: Sedimentary ancient DNA. Geobiology 9, 377–393 (2011).

2.   E. Rizzi, M. Lari, E. Gigli, G. De Bellis, D. Caramelli, Ancient DNA studies: new perspectives on old samples. Genet. Sel. Evol. 44, 21 (2012).

3.   A. Torti, M. A. Lever, B. B. Jørgensen, Origin, dynamics, and implications of extracellular DNA pools in marine sediments. Mar. Genomics 24, 185–196 (2015).

4.   E. Capo, et al., Tracking a century of changes in microbial eukaryotic diversity in lakes driven by nutrient enrichment and climate warming. Environ. Microbiol. 19, 2873–2892 (2017).

5.   K. G. Lloyd, B. J. MacGregor, A. Teske, Quantitative PCR methods for RNA and DNA in marine sediments: maximizing yield while overcoming inhibition. FEMS Microbiol. Ecol. 72, 143–151 (2010).

6.   V. Turon, E. Trably, A. Fayet, E. Fouilland, J.-P. Steyer, Raw dark fermentation effluent to support heterotrophic microalgae growth: microalgae successfully outcompete bacteria for acetate. Algal Res. 12, 119–125 (2015).

7.   L. Parducci, et al., Shotgun Environmental DNA, Pollen, and Macrofossil Analysis of Lateglacial Lake Sediments From Southern Sweden. Front. Ecol. Evol. 7, 189 (2019).

8.   Andrew. Ogram, G. S. Sayler, Denise. Gustin, R. J. Lewis, DNA adsorption to soils and sediments. Environ. Sci. Technol. 22, 982–984 (1988).

9.   A. Dell’Anno, S. Bompadre, R. Danovaro, Quantification, Base Composition, and Fate of Extracellular DNA in Marine Sediments. Limnol. Oceanogr. 47, 899–905 (2002).

10.   M. W. Pedersen, et al., Ancient and modern environmental DNA. Philos. Trans. R. Soc. B Biol. Sci. 370, 20130383 (2015).

11.   I. Domaizon, A. Winegardner, E. Capo, J. Gauthier, I. Gregory-Eaves, DNA-based methods in paleolimnology: new opportunities for investigating long-term dynamics of lacustrine biodiversity. J. Paleolimnol. 58, 1–21 (2017).

12.   C. Giguet-Covex, et al., New insights on lake sediment DNA from the catchment: importance of taphonomic and analytical issues on the record quality. Sci. Rep. 9, 14676 (2019).

13.   A. Vuillemin, et al., Preservation and Significance of Extracellular DNA in Ferruginous Sediments from Lake Towuti, Indonesia. Front. Microbiol. 8, 1440 (2017).

14.   E. Capo, I. Domaizon, D. Maier, D. Debroas, C. Bigler, To what extent is the DNA of microbial eukaryotes modified during burying into lake sediments? A repeat-coring approach on annually laminated sediments. J. Paleolimnol. 58, 479–495 (2017).

15.   E. Capo, D. Debroas, F. Arnaud, I. Domaizon, Is Planktonic Diversity Well Recorded in Sedimentary DNA? Toward the Reconstruction of Past Protistan Diversity. Microb. Ecol. 70, 865–875 (2015).

16.   I. Renberg, R. Bindler, M.-L. Brännvall, Using the historical atmospheric lead-deposition record as a chronological marker in sediment deposits in Europe. The Holocene 11, 511–516 (2001).

17.   M. Schwikowski, et al., Post-17th-Century Changes of European Lead Emissions Recorded in High-Altitude Alpine Snow and Ice. Environ. Sci. Technol. 38, 957–964 (2004).

18.   P. G. Appleby, F. Oldfield, The calculation of lead-210 dates assuming a constant rate of supply of unsupported 210Pb to the sediment. CATENA 5, 1–8 (1978).

19.   P. Guilizzoni, A. Marchetto, A. Lami, S. Gerli, S. Musazzi, Use of sedimentary pigments to infer past phosphorus concentration in lakes. J. Paleolimnol. 45, 433–445 (2011).

20.   B. Amann, et al., Spring temperature variability and eutrophication history inferred from sedimentary pigments in the varved sediments of Lake Żabińskie, north-eastern Poland, AD 1907–2008. Glob. Planet. Change 123, 86–96 (2014).

21.   S. M. Adl, et al., Revisions to the Classification, Nomenclature, and Diversity of Eukaryotes. J. Eukaryot. Microbiol. 66, 4–119 (2019).

22.   M. I. Love, W. Huber, S. Anders, Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014).